web/packages/teleport/src/Discover/Database/CreateDatabase/CreateDatabase.tsx

```diff
@@ -337,14 +337,15 @@ const CreateDatabaseDialog = ({
   );
 };
 
-// PORT_REGEXP only allows digits with length 4.
-export const PORT_REGEX = /^\d{4}$/;
+// only allows digits with valid port range 1-65535. Not using RegEx as more effienct to use logic comparsions.
 const requirePort = value => () => {
-  const isValidId = value.match(PORT_REGEX);
+  const numberValue = Number(value);
```

Comment:
You probably want a try/catch here, as `Number()` can throw.

Reply:
@hatched the parseInt should make the try/catch unnecessary right?